src/org/labkey/test/tests/flow/FlowSpecimenTest.java

```diff
@@ -92,6 +92,46 @@ public void _doTestSteps()
 
         // Issue 16945: flow specimen FK doesn't work for 'fake' FCS file wells created during FlowJo import
         verifyFlowDatasetSpecimenFK();
+
+        // Issue 48308: Flow: warn when deleting flow run with fcs files linked to study
+        verifyDeleteConfirmation();
+    }
+
+    private void verifyDeleteConfirmation()
+    {
+        log("** Attempt Specimen run delete, confirm usage before delete ");
+        goToFlowDashboard();
+        log("check that delete confirmation for unconnected files does not show Study linkage text");
+        clickAndWait(Locator.linkContainingText("FCS Files ("));
```

Comment:
` clickAndWait(Locator.linkContainingText("FCS Files"));`

Reply:
There are multiple links on the page that contain "FCS Files" and I want to target the one that looks like `FCS Files (X Runs)` but we may not know the actual number of runs. I can add a comment to this effect to make it clearer that the `(` is important and is intentional